packages/@ngtools/webpack/src/transformers/export_lazy_module_map.ts

```diff
@@ -52,10 +44,17 @@ export function exportLazyModuleMap(
   });
 
   const lazyModuleObjectLiteral = ts.createObjectLiteral(
-    modules.map((mod, idx) => ts.createPropertyAssignment(
-      ts.createLiteral(mod.loadChildrenString),
-      ts.createPropertyAccess(ts.createIdentifier(`__lazy_${idx}__`), mod.moduleName))
-    )
+    modules.map((mod, idx) => {
+      let [modulePath, moduleName] = mod.loadChildrenString.split('#');
+      if (modulePath.match(/\.ngfactory/)) {
+        modulePath = modulePath.replace('.ngfactory', '');
+        moduleName = moduleName.replace('NgFactory', '');
+      }
+
+      return ts.createPropertyAssignment(
```

Comment:
You can call ts.createUniqueName to create a unique Identifier. Might be better.

Reply:
The key is needed to match up with Universal.

The test would need updated to utilize Angular v5 and can be re-added once the CLI generates v5 applications.